### PR TITLE
Use: integration type: rephrase to make more procedural

### PR DESCRIPTION
**Pull request by c0ffeeca7**

### CODE REVIEW

File: source/docs/use/type/integration.md [hunk 1]
```diff
@@ -2,24 +2,26 @@
 title: Integration
 ---
 
-Integrations in HACS (or custom_components as they are often referred to) are community-built integrations.
+Integrations in HACS (also known as *custom components*) are community-built integrations.
```
Comment (ludeeus) [inline source/docs/use/type/integration.md:5]: Let's misuse abbreviations here for "integrations".

File: source/docs/use/dashboard.md [hunk 2]
```diff
@@ -12,9 +12,19 @@ This is based on the same data table that you will find elsewhere in the [Home A
...
-By default it only shows you which repositories you have downloaded with HACS.
+By default, it only shows you which repositories you have downloaded with HACS.
 
-If you want to see more use the [search](#search) and [filter](#filter) capabilities.
+If you want to see more, use the [search](#search) and [filter](#filter) capabilities.
+
+## Downloading a repository
+
+1. In Home Assistant, open HACS and find the repository of interest. 
+    - If you want to browse by type, [filter](/docs/use/dashboard.md#filter), for example for type *integration*.
```
Comment (c0ffeeca7) [inline source/docs/use/dashboard.md:22]:
```suggestion
    - If you are unsure what you are looking for and want to browse, you can [filter](/docs/use/dashboard.md#filter). For example for type *integration*.

```

File: source/docs/use/dashboard.md [hunk 3]
```diff
@@ -12,9 +12,20 @@ This is based on the same data table that you will find elsewhere in the [Home A
...
+
+## Downloading a repository
+
+1. In Home Assistant, open HACS and find the repository of interest. 
+    - If you are unsure what you are looking for and want to browse, you can [filter](/docs/use/dashboard.md#filter). For example for type *integration*.
+
+    - If you already know what you want, type the name into the search field.
+2. Select the repository from the list and read its documentation.
+3. To download the repository, select **Download**.
+    - The repository is now added to your [Home Assistant configuration directory](https://www.home-assistant.io/docs/configuration.md#to-find-the-configuration-directory).
```
Comment (ludeeus) [inline source/docs/use/dashboard.md:27]:
```suggestion
    - The files provided by the repository is now added to your [Home Assistant configuration directory](https://www.home-assistant.io/docs/configuration.md#to-find-the-configuration-directory).
```
You are not getting the entire repo, only the important parts.

File: source/docs/use/dashboard.md [hunk 4]
```diff
@@ -41,7 +52,7 @@ You can search data in all [columns](#columns) of each repository including:
 
 ## Filter
 
-If you are just looking to bowse repositories, you can do so with filtering.
+If you are just looking to browse repositories, you can do so with filtering.
```
Comment (ludeeus) [inline source/docs/use/dashboard.md:55]: image

File: source/docs/use/dashboard.md [hunk 5]
```diff
@@ -12,9 +12,20 @@ This is based on the same data table that you will find elsewhere in the [Home A
...
+By default, it only shows the repositories you have downloaded with HACS.
 
-If you want to see more use the [search](#search) and [filter](#filter) capabilities.
+If you want to see more, use the [search](#search) and [filter](#filter) capabilities.
+
+## Downloading a repository
+
+1. In Home Assistant, open HACS and find the repository of interest.
+    - If you are unsure what you are looking for and want to browse, you can [filter](/docs/use/dashboard.md#filter). For example, for type *integration*.
+
```
Comment (ludeeus) [inline source/docs/use/dashboard.md:23]:
```suggestion
```

**Review by ludeeus — APPROVED** [review]

**Pull request merged**